src/components/Input/Input.styled.tsx

```diff
@@ -6,8 +6,20 @@ import { pxToRem } from 'utils'
 import Box from '../Box/Box'
 import Typography from '../Typography/Typography'
 
+const getTextColor = (props) => {
+  if (props.$hasError) {
+    return props.theme.colors.utility.danger['100']
+  }
+
+  if (props.$disabled) {
+    return props.theme.colors.neutral.primary['30']
+  }
+
+  return ''
```

Comment:
Hm. So the css would be `color: ;`?

Don't we have a default color?

Reply:
no worries @vitalyrudenko-affinidi 
it will use the default color which implemented in `Typograpgy` component
here it will apply new color if it is disabled or has an error